src/components/dev-hub/feedback/feedback-container.js

```diff
@@ -11,21 +24,83 @@ const feedbackItems = graphql`
     }
 `;
 
-const FeedbackContainer = () => {
+const StyledButtonContainer = styled('div')`
+    display: flex;
+    justify-content: center;
+    margin-bottom: 4px;
+`;
+
+const StyledForm = styled('form')`
+    padding: 0 ${size.medium};
+
+    button,
+    > label,
+    ${StyledInputContainer} {
+        margin: ${size.default} 0;
+    }
+`;
```

Comment:
This is great, we can also just apply this directly to the `CMSForm` for now if we want to keep CSS clean.